libs/blocks/chart/chart.css

```diff
@@ -3,6 +3,34 @@
   to { transform: scale(1); }
 }
 
+.section.chart-section {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: flex-start;
+    margin-left: auto!important;
```

Comment:
Why the `!important`?  Overriding echarts css?

Reply:
Actually these aren't needed, copied them over on accident. Thanks.